src/app/profile/settings/connected-accounts/connected-accounts.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container-fluid container-cards-pf">
+  <div class="row-cards-pf accounts-component">
+    <div class="col-xs-10 col-sm-4 col-md-4">
+      <div class="card-pf">
+        <div class="card-pf-heading">
+          <div class="row account-content">
+            <div class="col-xs-6 col-sm-6 col-md-6">
+              <h2 class="card-pf-title">Connected Accounts</h2>
+            </div>
+            <div class="col-xs-6 col-sm-6 col-md-6">
+              <button class="btn btn-link pull-right" style="padding-right: 0" type="button" (click)="refreshServices()">
+                Refresh
+                <span class="fa fa-refresh"></span>
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-pf-body">
+          <div class="row account-content github-account-info">
+            <div class="col-xs-2 col-sm-2 col-md-2">
+              <i class="fa fa-github" style="font-size: 3em"></i>
+            </div>
+            <div class="account-text-box col-xs-6 col-sm-6 col-md-6">
+              <b>GitHub</b>
+              <br>
+              <span class="word-wrap" *ngIf="gitHubLinked; else errorName">{{gitHubUserName}}</span>
+              <ng-template #errorName><span>{{gitHubError}}</span></ng-template>
+            </div>
+            <div class="col-xs-4 col-sm-4 col-md-4">
+              <button *ngIf="gitHubLinked; else connectButton" class="btn btn-default pull-right" (click)="disconnectGitHub()">Disconnect</button>
+              <ng-template #connectButton>
+                <button *ngIf="!gitHubLinked" class="btn btn-primary pull-right" (click)="connectGitHub()">Connect Account</button>
+              </ng-template>
+            </div>
+          </div>
+          <div class="row account-content">
+            <div class="col-xs-2 col-sm-2 col-md-2">
+              <img class="icon" height="36px" width="42px" src="../../../../assets/images/openshift_icon.svg">
+            </div>
+            <div class="account-text-box col-xs-6 col-sm-6 col-md-6">
+              <b>OpenShift</b>
+              <br>
```

Comment:
avoid the use of `<br>` if possible - adding a class and bottom margin to `<b>OpenShift</b>` would suffice.